src/containers/ReviewProblemStepsModal/components/ReviewProblemStepsContent/components/AssessmentsTable/messages.js

```diff
@@ -18,7 +18,7 @@ const messages = defineMessages({
   },
   idAssessmentColumnTitle: {
     id: 'ora-grading.ReviewProblemStepsContent.idAssessmentColumnTitle',
-    defaultMessage: 'ID Assessment',
+    defaultMessage: 'Assessment ID',
```

Comment:
Why are you changing this? The mockup in the [Figma](https://www.figma.com/proto/dbUxY1HO4OoPtzxxyae1Xe/Status-progress-table-for-course-staff?type=design&node-id=31-1628&t=s7E27UO7p0G8FoEV-1&scaling=min-zoom&page-id=0%3A1&starting-point-node-id=17%3A5207) doesn't look like this

Reply:
From the API it is returned as `assessment_id`, I modified it for consistency. But it could be left as it looks in the mockup.